slashCommands/vote.ts

```diff
@@ -43,9 +58,16 @@ export default {
         const time = options.getInteger("votetime", false);
         const timed = time != undefined;
         const title = options.getString("name", true);
+        let maingroup = options.getRole("votegroup", false) as Role;
+        if (maingroup == undefined) {
+            const guid = get("vote_group", "config") as string;
+            maingroup = (guid == "")? interaction.guild.roles.everyone: interaction.guild.roles.cache.get(guid);
+        }
 
-        const guid = get("vote_group", "config") as string;
-        const maingroup = interaction.guild.roles.cache.get(guid);
+        if (membercanstartvote(interaction.member as GuildMember, maingroup)) {
```

Comment:
This fails when the role is `@everyone` since that role doesn't have an id and `membercanstartvote` tries to access it... I think we should check for everyone role here (by comparison) and only execute the function if the role isnt everyone
(e.g. `!maingroup.equals(everyone) && membercanstartvote`)

Reply:
the snowflake should be created at server creation? The id of a role isnt a optional so @everyone has to have one

Reply:
and it works just fine on my instance

Reply:
You're right, specifying everyone as the vote-group works, it didn't work for me when the `vote_group` isn't set at all in the config. This is actually partially a problem with the underlying configmanager, because instead of returning an empty string or `undefined` when trying to look up an object that doesn't exist, we currently return `{}`.
As a temporary workaround until I fix the config manager, we can change the condition in Line 64 to `JSON.stringify(guid) == "{}"`.